website/docs/api/runtime/schema.mdx

```diff
@@ -203,25 +209,33 @@ type CeramicAccount implements Node {
   # The following fields are always present
   id: ID!
   isViewer: Boolean!
-  # The Post relation is a connection because the Post account relation is LIST
-  postList: PostConnection
-  # The Post relation is a single object because the Profile account relation is SINGLE
+  # The Meeting relation is a connection because the Meeting account relation is LIST
+  # The meetingList connection allows to access all the Meeting documents controlled by the account
+  meetingList: MeetingConnection
+  meetingListCount: Int!
+  # The otherOfMeetingList connection allows to access all the Meeting documents where the account DID is set as the value of the "other" field
+  otherOfMeetingList: MeetingConnection
```

Comment:
are these field names auto-generated?  So it's always `<field name of relation field>Of<model name of model with relation>Connection`?

That feels quite prescriptive to me, but I also can't see a way around if we want these accessors to be autogenerated...

In the future, if users don't like these autogenerated names, could we add an `@` directive to the composite to specify what the name of the autogenerated accessors should be?

Reply:
Yes it's possible to alias fields when performing queries, it's supported directly by GraphQL